src/components/Books.js

```diff
@@ -1,22 +1,10 @@
-import { useSelector } from 'react-redux';
```

Comment:
- [ ] Kindly style your project to look identical to this template as demanded by the project requirements. Kindly ensure that all components on the template are added as placeholders without any added functionality and also ensure that the functionalities of the bookstore are retained.
![image](https://user-images.githubusercontent.com/94551484/224135303-c48ad885-0746-49bf-b1dc-9252e0adc5c8.png)